source/js/header-navigation.js

```diff
@@ -65,10 +65,10 @@ function toggleLeftShift() {
 
 /* START - Utility functions */
 function isElementInViewportYAxis(element) {
-  const rect = element.getBoundingClientRect();
+  var rect = element.getBoundingClientRect();
 
-  let verticalCheck = true;
-  if (window.innerWidth < globalVariables.breakpoints.lg) { // Tablet
+  var verticalCheck = true;
+  if (window.innerWidth < globalVariables.breakpoints.lg) { // Tabvar
```

Comment:
Accidentally renamed the Tab'let' to Tab'var' 💃

Reply:
Refactoring done right :D